### PR TITLE
feat: tailwind module integration

### PR DESCRIPTION
**Pull request by ineshbose**

### CODE REVIEW

File: src/module.ts [hunk 1]
```diff
@@ -65,6 +65,11 @@ export default defineNuxtModule({
       config.plugins.push(resolve(runtimeDir, 'nitro-plugin'))
     })
 
+    nuxt.hook('tailwindcss:config', (tailwindConfig) => {
+      options.classSuffix = ''
```
[the remark this answers is not in the document]
Comment (danielroe) [reply src/module.ts:69]: perhaps would be worth adding a test to this repo to confirm things work?

File: src/script.js
Comment (ineshbose) [inline src/script.js:1]: these updates don't seem to be playing well with a node worker - can revert the changes if necessary

File: src/module.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (ineshbose) [reply src/module.ts]: updated hook logic and added a demo on the playground 👍

File: docs/content/1.index.md [hunk 2]
```diff
@@ -186,7 +186,10 @@ Then in your `nuxt.config.ts`, set the `classSuffix` option to an empty string:
 
 ```ts [nuxt.config.ts]
 export default defineNuxtConfig({
-  modules: ['@nuxtjs/color-mode'],
+  modules: [
+    '@nuxtjs/color-mode',
+    // '@nuxtjs/tailwindcss' // specify after
```
Comment (danielroe) [inline docs/content/1.index.md:191]: ideally we wouldn't have to specify an order of these modules.

raised https://github.com/nuxt-modules/tailwindcss/issues/826 to discuss and resolve upstream in tailwindcss module

Comment (ineshbose) [reply docs/content/1.index.md]: @danielroe I've updated the docs to *not* have a dedicated section for integration with `@nuxtjs/tailwindcss` as there is nothing required to do anymore. The hook ordering we'll track in the tailwind module!

**Review by danielroe — APPROVED** [review]
